packages/pages/src/dev/server/public/templateBase.html

```diff
@@ -4,24 +4,23 @@
   <body>
     <div id="reactele"><!--app-html--></div>
     <script type="module">
```

Comment:
Yup, it's a copy paste! Noted in the pr description: changes in templateBase.html is copied over from running `pnpm pack` and use the content of the `compiled entry-client.js`. @mkilpatrick informed me we can't import entry-client.tsx directly because it doesn't work correctly for hmr. He opened a GH [issue](https://github.com/vitejs/vite/issues/9054) with Vite, which is not resolve yet. ([slack thread)](https://yext.slack.com/archives/C02LLE9BW2K/p1666193518715529?thread_ts=1666192140.834989&cid=C02LLE9BW2K)